client/app/scripts/superdesk-monitoring/monitoring.js

```diff
@@ -274,6 +274,7 @@
                 scope.$on('ingest:update', queryItems);
                 scope.$on('item:spike', queryItems);
                 scope.$on('item:unspike', queryItems);
+                scope.$on('item:mark', queryItems);
```

Comment:
it would be better to handle it in list item directive, without reloading the list every time item is marked for highlight